packages/core/test-project/ios/RNEmbraceTests/RNEmbraceTests.swift

```diff
@@ -266,7 +266,8 @@ class EmbraceSpansTests: XCTestCase {
         XCTAssertEqual(promise.rejectCalls[0], "Could not retrieve a span with the given id")
     }
 
-    func testAddSpanEvent() async throws {
+    // TODO fails on 6.3 currently due to span.flush call
```

Comment:
I think this accidentally got reverted with a merge, these tests can be unskipped now with the fix on 6.3

Reply:
done